refactor: added text and div tags to check-result

Kernix13 opened the pull request:
# Description

Added the text of `AA+` and `AAA+` to the `results-row` fields. I added 2 div tags and some CSS for the first div. It looks correct on small screens but I can not get the results row to display `flex-direction: row`. I also made comments in the styles.css file and grouped similar rules together. This is **not** a fix for issue 65.
Fixes # (65)

<!-- Please place an x in the [ ] to check off the type of change for this PR -->

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Chore items (this includes basic clean up of files, or updates to packages)
- [ ] Updates to documentation

<!-- Please make sure to go through the entire checklist before requesting a review. Place an x in the [ ] to check off all of the items completed -->

## Checklist

- [x] I have a descriptive title for my PR
- [x] I have linked this PR to the corresponding issue. [See how to do that here.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] I have run the project locally and reviewed the code changes
- [x] My changes generate no new warnings

AndreyDmitriev39r commented:
> This is not a fix for issue 65.
I don't understand one moment: do you want this branch to be merged or not? Please clarify.
If no, you can convert pull request to draft - more info about this in pinned messages in discord.

Also, you didn't request any reviewers. If you wanna feedback from someone about this changes, I suggest to add reviewers to this PR.

> I also made comments in the styles.css file and grouped similar rules together.

This is good change. But the thing is: we don't have issue for this. I thought normal workflow - it's when for every changes we have corresponding issues. And every pull request should address some existing issue.

> but I can not get the results row to display flex-direction: row

I will pull down this branch and will try to see what's what.

Kernix13 commented:
Why is "Fixes issue" in the template? I'm not done so it doesn't fix it. So every change needs an issue? Do we need an issue to add AA+ and AAA+? That will require additional div tags. Changes to the CSS are required as well. I didn't know I had to request reviewers. I thought people self-assigned themselves. 

What I'm planning on doing is to go through all the issues and discord discussions to make notes on everything I missed earlier in the week and there was a lot.

AndreyDmitriev39r commented:
> Why is "Fixes issue" in the template?

'fixes' is a keyword. It automatically connects your pull request with the issue.
So, when pull request will be merged >>> issue will be closed automatically by github.
You can also use keyword 'closes'

> So every change needs an issue?

Well, if we look at the issue #65 >>> there you mentioned AA and AAA stuff, right. So we - the rest of the team - are expecting, that somebody who is working on this issue(assignee I mean - in this case it's you) will make these changes.

I didn't saw in #65 any mentioning of grouping CSS rules, so in this PR I didn't expect such changes.

Issues and PRs need to be connected thematically.

Feel free to argue, the above for the discussion.

jdwilkin4 commented:
I added the WIP and Do not merge labels since you said you are still working on this.
In the future, if you are still working on an issue, then create a draft PR first.
That will tell people you are still working on it.
You can learn more about that here: 
https://github.com/DBenMoshe/contrast-ratio-repo/pull/28

Kernix13 commented:
I can set the CSS back to what it was but with the addition of the CSS for the elements I added. 

> I added the WIP and Do not merge labels since you said you are still working on this.

For some reason, the page is not displaying the way that the CSS is set up. I'm not getting `flex-direction: row;` for large screens for the `result-row`. I have no idea why it is doing that. There must be something bad on my end so when it comes to being done, I won't be. Maybe it's best that my PR is rejected. Thoughts?

> In the future, if you are still working on a PR, then create a draft PR first.

I did not know that was an option.
